requirements/constraints.txt

```diff
@@ -1,8 +1,8 @@
 #
-# This file is autogenerated by pip-compile with python 3.10
```

Comment:
The constraint file generation runtime must remain the same.

Reply:
I have regenerated it again now using Python 3.10

Reply:
Ack. Marking as resolved. Will review the updated thing.